internal/services/mssql/mssql_database_resource_test.go

```diff
@@ -535,7 +543,7 @@ func TestAccMsSqlDatabase_scaleReplicaSetWithFailovergroup(t *testing.T) {
 				check.That(data.ResourceName).Key("sku_name").HasValue("GP_Gen5_2"),
 			),
 		},
-		data.ImportStep(),
+		data.ImportStep(), // need to ignore create mode here
```

Comment:
Do we still need to ignore create mode here?

Reply:
That was a note to myself while I was testing it and I forgot to remove it.

Fixed.